dist.ini

```diff
@@ -36,6 +36,7 @@ Alien::Build = 0.27
 
 [Prereqs / BuildRequires]
 Alien::Build = 0.27
+File::Touch = 0
```

Comment:
Better to use the `requires` directive (see suggestion above) in the `alienfile` because we only need this for a `share` install (`Path::Tiny` as mentioned is already an indirect dep so that point is somewhat moot, but good practice).